assets/sass/patterns/molecules/about-profile.scss

```diff
@@ -1,16 +1,55 @@
 @import "../../definitions";
 
+.about-profile--popup {
+  @media only all and (min-width: #{$bkpt-site--medium}px) {
+    cursor: pointer;
+  }
+}
+
 .about-profile__name {
   @include teaser-header-text-style();
+
+  .popup & {
+    @include nospace();
+    @include popup_title_text();
+  }
 }
 
 .about-profile__role {
   font-style: italic;
   margin: 0;
+
+  .popup & {
+    @include popup-large-ancillary-text();
+    font-style: normal;
+  }
 }
 
 .about-profile__profile {
   @include margin(24, "top");
+
+  @media only all and (min-width: #{$bkpt-site--medium}px) {
+
+    .about-profile--popup & {
+      display: none;
+
+      .popup & {
+        display: block;
+        margin-top: 0;
+
+        & p,
```

Comment:
There's some pretty funky `&` use going on here. Bit confusing, should be simplified.